modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemsXlsMVCResourceCommand.java

```diff
@@ -0,0 +1,462 @@
+		}
+	}
+
+	private int _createFileHeaderCells(
+		Row headerRow, int headerRowCellIndex, Locale locale) {
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "description"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "extension"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "file-name"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow, LanguageUtil.get(locale, "size"));
+
+		return headerRowCellIndex;
+	}
+
+	private int _createFileSpecificDataCells(
+		int cellIndex, Locale locale, ResourceResponse resourceResponse,
+		ResourceRequest resourceRequest, Row row, ThemeDisplay themeDisplay,
+		ContentDashboardItem<?> contentDashboardItem) {
+
+		JSONObject fileSpecificDataJSONObject =
+			contentDashboardItem.getSpecificInformationJSONObject(
+				ParamUtil.getString(resourceRequest, "backURL"),
+				_portal.getLiferayPortletResponse(resourceResponse), locale,
+				themeDisplay);
+
+		if (fileSpecificDataJSONObject == null)
+
+			return cellIndex;
+
+		String fileDescription = fileSpecificDataJSONObject.get(
+			"description"
+		).toString();
+
+		cellIndex = _createCell(cellIndex, row, fileDescription);
+
+		String extension = fileSpecificDataJSONObject.get(
+			"extension"
+		).toString();
+
+		cellIndex = _createCell(cellIndex, row, extension);
+
+		String fileName = fileSpecificDataJSONObject.get(
+			"fileName"
+		).toString();
+
+		cellIndex = _createCell(cellIndex, row, fileName);
+
+		String size = fileSpecificDataJSONObject.get(
+			"size"
+		).toString();
+
+		return _createCell(cellIndex, row, size);
+	}
+
+	private void _createHeaderRow(
+		Locale locale, Sheet sheet, Workbook workbook) {
+
+		Row headerRow = sheet.createRow((short)0);
+
+		headerRow.setRowStyle(
+			_createCellStyle(true, "Helvetica", (short)11, workbook));
+
+		int headerRowCellIndex = _createBasicDataHeaderCells(
+			locale, 0, headerRow);
+
+		headerRowCellIndex = _createFileHeaderCells(
+			headerRow, headerRowCellIndex, locale);
+
+		_createJournalArticleHeaderRow(headerRow, headerRowCellIndex, locale);
+	}
+
+	private void _createJournalArticleHeaderRow(
+		Row headerRow, int headerRowCellIndex, Locale locale) {
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "display-date"));
+
+		headerRowCellIndex = _createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "creation-date"));
+
+		_createCell(
+			headerRowCellIndex, headerRow,
+			LanguageUtil.get(locale, "languages-translated-into"));
+	}
+
+	private void _createJournalArticleSpecificDataCells(
+		int cellIndex, Locale locale, ResourceResponse resourceResponse,
+		ResourceRequest resourceRequest, Row row, ThemeDisplay themeDisplay,
+		ContentDashboardItem<?> contentDashboardItem) {
+
+		JSONObject journalArticleSpecificDataJSONObject =
```

Comment:
Keep consistency between variable name and getter name.

Reply:
It is not a requirement right now but in the future some style will need to be applied,.. but sure, can be removed and added later